### PR TITLE
Bluespace locker

### PR DESCRIPTION
**Pull request by monster860**
https://streamable.com/puspt

I can't wait to see assistants shutting themselves in every locker they can find to try to find this thing.

:cl: monster860
add: Adds the bluespace locker. Every round, a random non-secure locker on the station becomes a bluespace locker. Bluespace lockers have entire rooms inside of them, and they are indistinguishable from normal lockers, except that when you go inside it you find yourself in a room. If you destroy the bluespace locker, it randomly replaces a different locker on the station.
/:cl:

### CODE REVIEW

**Comment by fluffe9911** [comment]
Neat

**Comment by Time-Green** [comment]
Literally the greatest feature ever conceived

File: yogstation/code/game/area/Space_Station_13_areas.dm [hunk 1]
```diff
@@ -19,4 +19,10 @@
...
 						 'sound/voice/lowHiss4.ogg',
-						 'yogstation/sound/misc/honk_echo_distant.ogg')
\ No newline at end of file
+						 'yogstation/sound/misc/honk_echo_distant.ogg')
+
+/area/bluespace_locker
+  name = "Bluespace Locker"
+  icon_state = "away"
+  requires_power = FALSE
+  has_gravity = STANDARD_GRAVITY
```
Comment (Time-Green) [inline yogstation/code/game/area/Space_Station_13_areas.dm:28]: Maybe noteleport? Might be bad if people end up free on the centcon zlevel

Comment (monster860) [reply yogstation/code/game/area/Space_Station_13_areas.dm:28]: I thought that just prevents wizards from teleporting to it

**Comment by Bawhoppen** [comment]
And they said Yogs would just become another tg clone...

**Comment by Amelia0010** [comment]
@Bawhoppen it is

File: yogstation/code/game/area/Space_Station_13_areas.dm (hunk 1 above)
Comment (Time-Green) [reply yogstation/code/game/area/Space_Station_13_areas.dm:28]: Shit you’re right. To be honest a wizard suddenly appearing from a locker would be pretty good

**Review by ThatLing — DISMISSED** [review]
On hold until we make YogsCentCom

**Comment by Time-Green** [comment]
yogs centcom is merged now SPEEDMERGE

**Review by Amelia0010 — DISMISSED** [review]
just move the map changes over to the yogsCC and everything is cool I thonk

**Comment by monster860** [comment]
there travis CI passed now *speedmerge*

**Pull request merged**